test/support/videoroom_test_helper.ex

```diff
@@ -0,0 +1,41 @@
+defmodule VideoRoomTest.Helper do
+  alias Janus.Plugin.VideoRoom.Errors
+
+  def error_message(error_code, videoroom \\ "event", error \\ "error message") do
+    %{
+      "error_code" => error_code,
+      "videoroom" => videoroom,
+      "error" => error
+    }
+  end
+
+  defmacro test_connection_error(fun, args) do
```

Comment:
this and following macro looks nigh identical.

Reply:
After latest commit they should slightly differ.